lib/Serialization/Serialization.cpp

```diff
@@ -1361,6 +1363,19 @@ void Serializer::writeGenericRequirements(ArrayRef<Requirement> requirements,
   }
 }
 
+void Serializer::writeInlinableBodyText(const AbstractFunctionDecl *AFD) {
+  using namespace decls_block;
+
+  if (AFD->getResilienceExpansion() != swift::ResilienceExpansion::Minimal)
+    return;
```

Comment:
@slavapestov, this is the correct check for inlinable functions of any kind, right?